doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,550 @@
+  * Added a separate workqueue for the application, instead of using the system workqueue.
+
+* :ref:`gps_with_supl_support_sample`:
+
+  * Added support for the new :ref:`supl_client` library, if enabled.
+
+
+Updated libraries
+-----------------
+
+* :ref:`lib_download_client`:
+
+  * Added the :option:`CONFIG_DOWNLOAD_CLIENT_MAX_TLS_FRAGMENT_SIZE` option that allows to configure fragment sizes for TLS connections and non-TLS connections independently.
+  * Added support for using non-default ports.
+
+* :ref:`lib_spm`:
+
+  * Updated the security attribution to configure the peripherals NRF_REGULATORS and NRF_WDT as Non-secure. ??? not updated in docs
+  * Added the RTC0 peripheral (as Non-Secure). ??? correct that this has been added and configured while the ones above have only been configured? Also, not updated in docs.
+  * Fixed a bug where the library attempted to set the IRQ target state of the P0 peripheral.
+
+* :ref:`lib_fota_download`:
+
+  * Added an optional progress event (:cpp:enumerator:`FOTA_DOWNLOAD_EVT_PROGRESS <fota_download::FOTA_DOWNLOAD_EVT_PROGRESS>`) that informs the user of the library how many bytes have been downloaded.
+  * Fixed a bug where the library continued downloading even if writing to the DFU target failed.
+  * Implemented a mechanism to retry downloads if a socket error occurs.
+
+* :ref:`lib_aws_fota`:
+
+  * Added functionality to resume jobs that are marked as being in progress, which ensures a more robust FOTA operation through AWS IoT jobs.
+  * Added offset reporting through the ``statusDetails`` field in an AWS IoT job, which makes it possible to track the progress of a FOTA operation more precisely.
+  * Removed the unused ``app_version`` parameter from the :cpp:func:`aws_fota_init` function.
+  * Inversed the interpretation of the return value of :cpp:func:`aws_fota_mqtt_evt_handler`.
+    0 now indicates success, and no further handling is required.
+    1 indicates that further processing is required by the :cpp:func:`mqtt_evt_handler` that called :cpp:func:`aws_fota_mqtt_evt_handler`.
+
+* :ref:`lib_nrf_cloud`:
+
+  * Removed the button/switch pairing method.
+  * Added functionality to handle the device configuration in the device shadow.
+
+* :ref:`liblwm2m_carrier_readme`:
+
+  * Updated to version 0.8.1.
+
+* Moved the following libraries from ``drivers/`` to ``lib/``:
+
+  * :ref:`at_cmd_readme`
+  * ``lte_link_control``
+
+
+Updated drivers
+---------------
+
+* Moved the following drivers from ``drivers/`` to ``drivers/gps/``:
+
+  * ``gps_sim``
+  * ``nrf9160_gps``
+
+
+BSD library
+-----------
+
+* Updated the :ref:`nrfxlib:bsdlib` to version 0.6.1.
+
+
+Common libraries, drivers, and samples
+======================================
+
+* Added the following sample:
+
+  * :ref:`radio_test` - demonstrates how to configure the radio in a specific mode and then test its performance.
+    This sample was ported from the nRF5 SDK.
+
```

Comment:
missing entry for ble mesh examples @trond-snekvik

Reply:
I discussed with @bjorn-tore-taraldsen and @MonikaLieLarsen yesterday and the decision was to leave Mesh out of the release notes.

Reply:
We will leave Mesh out of the scope for what's mention for this release.